libraries/classes/User.php

```diff
@@ -133,4 +133,14 @@ public static function isLoggedIn()
       return false;
     }
   }
+  public static function sendFeedback(){
+    $db = new Factory;
+    $mysql = $db->getDBO();
+
+    $name = $_POST['name'];
+    $email = $_POST['email'];
+    $message = $_POST['message'];
+    $sql = "INSERT INTO feedback (name,email,message) VALUES ('$name','$email','$message')";
```

Comment:
Check for MySQL error after insert. And send a proper response based on the result.

Reply:
Send proper response see other examples.